scripts/eurostat/gdp/import_data.py

```diff
@@ -81,9 +86,23 @@ def clean_data(self):
                              "Please check you are calling preprocess_data "
                              "before clean_data.")
         self.clean_df = self.preprocessed_df[self.DESIRED_COLUMNS]
+        self.clean_df = self.clean_df[~self.clean_df['geo'].isin(['EU27_2020',
+                                                                  'EU28'])]
+
+        def geo_converter(geo):
+            """Converts geo codes to nuts or country codes."""
+            numbers = "0123456789"
+            if any(num in geo for num in numbers):
+                return 'nuts/' + geo
+            elif geo in COUNTRY_MAP:
+                return COUNTRY_MAP[geo]
+            else:
+                return ""
 
         # Prepends nuts/ prefix to geo codes.
-        self.clean_df['geo'] = self.clean_df['geo'].apply(lambda g: "nuts/" + g)
+        self.clean_df['geo'] = self.clean_df['geo'].apply(geo_converter)
+        # Remove geos that include all european countries
```

Comment:
```suggestion
        # Remove geos that include all European countries.
```

Reply:
I realized this was an old comment. I changed it to
```
# Remove geos that do not adjust to any of the recognized standards.
```